src/features/TimeframeScroll.ts

```diff
@@ -1,27 +1,31 @@
 class TimeframeScroll extends Feature {
+  // Separating hotkeys into individual variables
   triggerDown = false;
-  
+  hotkey = 'Tab';
+  ctrl = false;
+  shift = false;
+  alt = false;
+  meta = false;
+
   constructor() {
     super(
       'Scroll Timeframes',
       'Allows you to scroll timeframes using a modifier + scroll wheel',
       true,
       {
-        key: 'Tab',
-        ctrl: false,
-        shift: false,
-        alt: false,
-        meta: false
+        key: this.hotkey,
+        ctrl: this.ctrl,
+        shift: this.shift,
+        alt: this.alt,
+        meta: this.meta
```

Comment:
I'm getting the warning that "super must be called before accessing 'this' in the constructor of a derived class." meaning that this change breaks a few things.  I think it would be better to just revert it altogether.

Reply:
Yeah, this was a bad take on my part.  I have reverted it and tweaked it.